docs/compression_algorithms/Pruning.md

```diff
@@ -90,7 +90,7 @@ sparsity and filter pruning algorithms. It can be enabled by setting a non-zero
         "prune_first_conv": false, // Whether to prune first Convolutional layers or not. First means that it is a convolutional layer such that there is a path from model input to this layer such that there are no other convolution operations on it. `False` by default. 
         "prune_last_conv": false, // Whether to prune last Convolutional layers or not.  Last means that it is a Convolutional layer such that there is a path from this layer to the model output such that there are no other convolution operations on it. `False` by default. 
         "prune_downsample_convs": false, // Whether to prune downsample Convolutional layers (with stride > 1) or not. `False` by default.
-        "prune_batch_norms": false, // Whether to nullifies parameters of Batch Norm layer corresponds to zeroed filters of convolution corresponding to this Batch Norm. `False` by default.
+        "prune_batch_norms": true, // Whether to nullifies parameters of Batch Norm layer corresponds to zeroed filters of convolution corresponding to this Batch Norm. `True` by default.
```

Comment:
@AlexKoff88 Are we still need `prune_batch_norms `as a parameter? It looks like without the `prune_batch_norms=True` model will be pruned incorrectly after training (non-zero BatchNorm params will be deleted).